### ISSUE TITLE
use_jwt_access_tokens option

### ISSUE THREAD

**Issue by taat** [API-Skeletons/zf-oauth2-doctrine#86]
I've implemented authorization using Doctrine and it works fine if the 
`'use_jwt_access_tokens'`  option is set to false.

When I enable `'use_jwt_access_tokens' => true` during authentication I get access_tokens and refresh_tokens as JTW, but during authorization they don't work. If the JWT token is not the same as access_token, how do I get it working?

I thought the workflow would be the same, but instead the shorter random string I'll get detailed, encrypted JWT.

Here is a detailed question:
https://stackoverflow.com/questions/48917174/how-to-access-protected-resource-in-apigility-using-jwt

In another issue #29 regarding JWT you mentioned this article:
http://tomhanderson.blogspot.com/2015/05/jwt-in-oauth2-with-apigility-and.html
but it's not accessible anymore.

**Comment by taat** [comment]
Now I can see that it fails on JWT verification in:

      \OAuth2\Encryption\Jwt::verifySignature($signature, $input, $key, $algo = 'HS256')

Why the tokens returned by `/oauth` are invalid?

**Comment by taat** [comment]
This was caused by wrong default encryption algorithm I assumed.

I assumed that the default is HS256, but the default in OAuth2 library is RS256.

So I needed to store in the database public key and the default algorithm RS256.

Also I needed to use  ` 'store_encrypted_token_string' => false` .

These are both just options, which were not clear for me, not bugs, and related to OAuth2 library, not to this repo. So closing this one now, sorry for the hassle. Maybe this helps someone someday too.
